.github/workflows/validate-frontmatter.yml

```diff
@@ -44,12 +45,20 @@ jobs:
             npm install --silent
             node frontmatter-validator.js '${{ env.changed }}'
 
-      # Step 5: Comment on the PR if the previous step failed
-      - name: Comment on PR
+      # Step 5: Write workflow job summary if the previous step failed
+      - name: Add Job Summary
         if: failure()
+        run: |
+          echo "Hi @$pr_owner," >> $GITHUB_STEP_SUMMARY
+          echo "${{ steps.check_markdown.outputs.stdout }}" >> $GITHUB_STEP_SUMMARY
+
+      # Step 6: Leave a comment if the validation has failed and if it's not a PR from a fork
+      - name: Comment on PR
+        if: failure() && github.repository != 'SSWConsulting/SSW.Rules.Content'
```

Comment:
I think this logic is backwards
It says it will leave a comment if its **not** 'SSWConsulting/SSW.Rules.Content'

Reply:
Yes the code logic was unclear here. I meant to leave a comment only when it's not a fork. Thanks for pointing it out. I've replaced the hardcode with `github.event.pull_request.head.repo.fork == false`  in the latest commit